skycatalogs/objects/gaia_object.py

```diff
@@ -143,6 +141,8 @@ def get_config(cls):
     @ignore_erfa_warnings
     @staticmethod
     def load_collection(region, skycatalog, mjd=None):
+        if not skycatalog:
```

Comment:
I don't think this test is necessary or appropriate, given how these classes are structured.  To do it right, it would need to be
```
if not isinstance(skycatalog, SkyCatalog):
```
but then you'd need to import `SkyCatalog` from the other module and then have a circular dependency.    This is a public method, so in this case, I think it would be better just to document the parameters with a docstring. Also, it's not as if every function in this package tests if all of their passed parameters have the right type.

Reply:
True, but it will break if skycatalog is None.   If you think documenting it is sufficient I'll go with that.

> Also, it's not as if every function in this package tests if all of their passed parameters have the right type.

All too true.